apps/prairielearn/src/pages/partials/navbarInstitution.ejs

```diff
@@ -1,11 +1,11 @@
 <li class="nav-item">
-  <a class="nav-link" href="/pl/institution/<%= institution.id %>/admin">
-    <%= institution.short_name %>
+  <a class="nav-link" href="/pl/administrator/institutions">
+    Admin
```

Comment:
Minor change: the "Admin" navbar item now links back to the global admin "institutions" page. This makes it possible to jump back there without having to go all the way home. There was really no reason to have two links that both went to the same place.